File: sdk/csharp/examples/QuickProfileV3.cs
using System;
using System.Net.Http;
using System.Text;
using System.Threading.Tasks;

var host = Environment.GetEnvironmentVariable("MLX_LAUNCHER_HOST") ?? "launcher.mlx.yt";
var port = Environment.GetEnvironmentVariable("MLX_LAUNCHER_PORT") ?? "45001";
var url = $"https://{host}:{port}/api/v3/profile/quick";

var json = """
{
  "browser_type": "mimic",
  "core_version": 124,
  "os_type": "linux",
  "automation": "selenium",
  "is_headless": false,
  "parameters": {
    "flags": { "navigator_masking": "custom", "proxy_masking": "custom" },
    "proxy": { "host": "host.example", "type": "http", "port": 8080 }
  }
}
""";

using var client = new HttpClient();
var request = new HttpRequestMessage(HttpMethod.Post, url);
request.Headers.Add("Accept", "application/json");
request.Content = new StringContent(json, Encoding.UTF8, "application/json");

var response = await client.SendAsync(request);
response.EnsureSuccessStatusCode();
Console.WriteLine(await response.Content.ReadAsStringAsync());

File: sdk/java/examples/QuickProfileV3.java
// Multilogin X — Quick profile v3 (OkHttp)
import okhttp3.MediaType;
import okhttp3.OkHttpClient;
import okhttp3.Request;
import okhttp3.RequestBody;
import okhttp3.Response;

public class QuickProfileV3 {
    private static final MediaType JSON = MediaType.parse("application/json");

    public static void main(String[] args) throws Exception {
        String host = env("MLX_LAUNCHER_HOST", "launcher.mlx.yt");
        String port = env("MLX_LAUNCHER_PORT", "45001");
        String url = String.format("https://%s:%s/api/v3/profile/quick", host, port);

        String json = """
            {
              "browser_type": "mimic",
              "core_version": 124,
              "os_type": "linux",
              "automation": "selenium",
              "is_headless": false,
              "parameters": {
                "flags": { "navigator_masking": "custom", "proxy_masking": "custom" },
                "proxy": { "host": "host.example", "type": "http", "port": 8080 }
              }
            }
            """;

        Request request = new Request.Builder()
            .url(url)
            .post(RequestBody.create(json, JSON))
            .addHeader("Accept", "application/json")
            .build();

        try (Response response = new OkHttpClient().newCall(request).execute()) {
            if (!response.isSuccessful()) {
                throw new RuntimeException("HTTP " + response.code());
            }
            System.out.println(response.body().string());
        }
    }

    private static String env(String key, String fallback) {
        String v = System.getenv(key);
        return v != null && !v.isBlank() ? v : fallback;
    }
}
